org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -816,6 +820,7 @@ public Preferences() {
 		avoidVolatileChanges = true;
 		nonnullTypes = new ArrayList<>();
 		nullableTypes = new ArrayList<>();
+		nullAnalysisMode = FeatureStatus.interactive;
```

Comment:
Make the initial value to be disabled on server side. Let the client to determine whether to adopt it.